app/fantasy_cricket/team.py

```diff
@@ -0,0 +1,236 @@
+        """
+        Returns max player for a particular role
+        :param role_dict : dict of the players mapped to the score for that role
+        :type dict( str : float)
+        :param player : dictionary of the selected players for the best11
+        :type dict( str: float)
+        :param role : role of the player
+        :type str
+        :rtype: dict( str: float)
+        """
+        if role_dict!={}:
+            max_score = {
+                "wk": [list(role_dict.keys())[0]],
+                "all": [list(role_dict.keys())[0]],
+                "bat": [key for key in role_dict][:3],
+                "bowl": [key for key in role_dict][:3],
+            }
+
+            names = max_score[role]
+            for name in names:
+                if list(self.team_dict.keys())[0] in role_dict[name]["team"]:
+                    self.team_dict[list(self.team_dict.keys())[0]] += 1
+                elif list(self.team_dict.keys())[1] in role_dict[name]["team"]:
+                    self.team_dict[list(self.team_dict.keys())[1]] += 1
+                if name not in self.player:
+                    self.player[name] = role_dict[name]
+
+    def get_restofteam(self, role_dict, reduntant, restteam):
+        """
+        Returns the rest of the players who have not yet been chosen for the 11
+        :param role_dict : dict of the players mapped to the score for that role
+        :type dict( str : float)
+        :param player : dictionary of the selected players for the best11
+        :type dict( str: float)
+        :param redundant : selected players
+        :type list(str)
+        :param restteam: dict of the players left to be selected
+        :type dict( str: float)
+        :rtype dict( str: float)
+        """
+        for i in role_dict:
+            if i not in self.player and i not in reduntant:
+                restteam[i] = role_dict[i]
+                
+        restteam = sorted(restteam.items(), key=lambda x: x[1]["score"], reverse=True)
+        restteam = {i[0]: i[1] for i in restteam}
+        return restteam
+
+    def get_captain(self):
+        """
+        Returns Captain and Vice Captain for the selected 11
+        :param player: dictionary of the selected players for the best11
+        :type dict( str : float)
+        :rtype: tuple(str,str)
+        """
+        self.player = [ i[0] for i in sorted(self.player.items(), key=lambda x: x[1]["score"], reverse=True)]
+        captain, vcaptain = self.player[0], self.player[1]
+        return captain, vcaptain
+
+    def team(self):
+        """
+        Returns Captain,Vicecaptain and teams for each role
+        :rtype: tuple(
+                    str,
+                    str,
+                    dict( str : float),
+                    dict( str : float),
+                    dict( str : float),
+                    dict( str : float),
+                )
+        """
+
+        file_handler = open(self.path)
+        self.data = json.load(file_handler)
```

Comment:
This can be done as `json.load(open(self.path))`, specially since you aren't closing the file later

Reply:
Aah right lesser variables